### DISCUSSION TITLE
Running GPTH second time

### DISCUSSION THREAD

**Discussion by Amotsoko** [TheLastGimbus/GooglePhotosTakeoutHelper#254] [I NEED HELP]
Hi

I do not have previous experience with GPTH.
I tried to transfer my photos from Google Photos to my new Synology NAS. I made the mistake of not putting all the photos in one folder (I thought having sub-folders was okay), but that did not work of course.
When trying the second time I get this message:
![2023-10-13 09_12_39-C__Users_Amots_Downloads_gpth-windoza exe](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/50144878/21fa30cb-95ff-4853-b321-4b1744c90b01)

I am not sure what to do. Can you please assist (in simple, non-technical words).
Thanks

**Comment by TheLastGimbus** [comment]
Do not run gpth multiple times on same folder. I just did a broad explenation few days ago: https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/issues/253#issuecomment-1754558791

You need to unzip everything again

**Reply by Amotsoko** [reply]
Dear Mateusz Soszyński

Just to make sure, I deleted everything and downloaded it again and unzipped it again, but it is giving me the same message above.
I have unzipped it to the same drive (D:), but not the same folder so I am not sure what I am doing wrong?
I am not the most technical person unfortunately, but I would like to make this work.

Looking forward to your reply
Thanks

**Reply by TheLastGimbus** [reply]
Did you maybe solve it by this time?

If not, make sure you have clean nice unzipped folder, you are selecting the input folder correctly, and **there are** actual photos in this folder :eyes:

**Comment by Amotsoko** [comment]
Dear Mateusz

The problem was not solved!
I deleted everything, downloaded all the photos again, unzipped them again,
put them all in one folder in a different location than the first one.
The mistake is still there. I gave up as i did not get any answers from you
or from Synology.

**Reply by TheLastGimbus** [reply]
As you opened a second discussion (why??), i will lock this one and reply to evertything on new one
